### PR TITLE
[Homework #19] - react-api by German Macchia

### PR DESCRIPTION
**Pull request by GermanMacchia**

### CODE REVIEW

**Comment by GermanMacchia** [comment]
Hi @VitalikMelnyk, 
I leave my first approach for react-api task
Let me know for any suggestions!

File: homeworks/german-macchia/19-react-api/calculator-app/src/components/Keypad.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
+import { Button } from "../components/Button";
+import deleteIcon from "../assets/delete_icon.svg";
+import divideIcon from "../assets/divide_icon.svg";
+import minusIcon from "../assets/minus_icon.svg";
+import plusIcon from "../assets/plus_icon.svg";
+import xIcon from "../assets/x_icon.svg";
+import equalIcon from "../assets/equal_icon.svg";
+import React from "react";
+
+export const Keypad = React.memo(({ handleOnPushKey }) => {
```
Comment (VitalikMelnyk) [inline homeworks/german-macchia/19-react-api/calculator-app/src/components/Keypad.js:10]: A lot of redundant code.
You can create a configuration array of objects with settings about each button and map this array.
For example:
```
 const array = [
  {
    id: 0,
    innerHtml: <img src={deleteIcon} alt="delete icon" />,
    boxStyle: 'button border-box corner--left',
    value: 'C',
  },
];
```

File: homeworks/german-macchia/19-react-api/calculator-app/src/components/Keypad.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
+import { Button } from "../components/Button";
```
Comment (VitalikMelnyk) [inline homeworks/german-macchia/19-react-api/calculator-app/src/components/Keypad.js:1]: Format code with Prettier.

Comment (GermanMacchia) [reply homeworks/german-macchia/19-react-api/calculator-app/src/components/Keypad.js:1]: done!

File: homeworks/german-macchia/19-react-api/calculator-app/src/components/Keypad.js (hunk 1 above)
Comment (GermanMacchia) [reply homeworks/german-macchia/19-react-api/calculator-app/src/components/Keypad.js:5]: @VitalikMelnyk I've already upload a new commit with the changes you suggest. thanks

**Review by VitalikMelnyk — APPROVED** [review]

**Pull request merged**